### PR TITLE
Key Vault: moved parameters users can't set to kwargs

### PR DESCRIPTION
**Pull request by iscai-msft**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/models.py [hunk 1]
```diff
@@ -255,23 +253,20 @@ class Certificate(object):
...
     """
 
     def __init__(
         self,
         policy,  # type: CertificatePolicy
         properties=None,  # type: Optional[CertificateProperties]
-        key_id=None,  # type: Optional[str]
-        secret_id=None,  # type: Optional[str]
         cer=None,  # type: Optional[bytes]
+        **kwargs  # type: ANy
```
[the remark this answers is not in the document]
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/models.py]: thanks!

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/models.py [hunk 2]
```diff
@@ -116,16 +116,14 @@ def inner_error(self):
...
 
-    def __init__(self, attributes=None, cert_id=None, thumbprint=None, **kwargs):
-        # type: (Optional[models.CertificateAttributes], Optional[str], Optional[bytes], **Any) -> None
-        self._attributes = attributes
-        self._id = cert_id
-        self._vault_id = parse_vault_id(cert_id)
-        self._thumbprint = thumbprint
+    def __init__(self, **kwargs):
+        # type: (**Any) -> None
+        self._attributes = kwargs.get('attributes', None)
```
[the remark this answers is not in the document]
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/models.py]: ran it and it caught some things in models, thanks!

**Pull request merged**